kaggle: add page (#3901)
tldr-pages/tldr: pages/common/kaggle.md

# kaggle

> > Official API for kaggle accessible using a command line tool implemented in Python 3.

Reviewer: ```suggestion
> Official API for Kaggle via a command line tool implemented in Python 3.
```

Author: ```suggestion
> Official API for Kaggle. The API is accessible via a command line tool implemented in Python 3.
```

> > Official API for Kaggle. The API is accessible via a command line tool implemented in Python 3.

Reviewer 2: Perhaps this will read better?
```suggestion
> Official CLI for Kaggle implemented in Python 3.
```

> > More information: <https://www.kaggle.com>.

Reviewer 3: Is any kaggle cli document we can put here? Not just the home page of official site? I've found [this](https://github.com/Kaggle/kaggle-api). 

```suggestion
> More information: <https://github.com/Kaggle/kaggle-api>.
```
I suggest we can put the direct document link here.

- View current configuration values:

`kaggle config view`

> - Download file from competition dataset:

Reviewer 3: ```suggestion
- Download specific file from competition dataset:
```
I found

```
  -f FILE_NAME, --file FILE_NAME
                        File name, all files downloaded if not provided
```

> - Download specific file from competition dataset:

Reviewer 2: Perhaps this will read better?
```suggestion
- Download a specific file from a competition dataset:
```
